### PR TITLE
Create Android Github Action

### PR DESCRIPTION
**Pull request by sanchezz93**
Creating the Android github action -
This will set the Java version to '8' in the action and will run "gradlew build"
Solves #413 for Android

### CODE REVIEW

**Comment by tyvsmith** [comment]
Please ensure Fossa runs.

File: .github/workflows/android.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        java-version: ${{ matrix.java_version }}
+    - name: Gradle Wrapper Validation
+      uses: gradle/wrapper-validation-action@v1
+    - name: Configure Gradle
+      # Gradle configuration install deps 
+      run: ./gradlew help
+    - name: Spot check
+      run: ./gradlew spotlessCheck --stacktrace
+    - name: Build Project
+      run: ./gradlew assemble --stacktrace
```
Comment (jbarr21) [inline .github/workflows/android.yml:31]: is it better to split these steps out into assemble/test/check or just used `build` like [travis setup](https://github.com/uber/RIBs/blob/master/.travis.yml#L24) did?

**Comment by jbarr21** [comment]
> Please ensure Fossa runs.

looks like Fossa was [never setup](https://github.com/uber/RIBs/blob/master/.travis.yml#L24). i suggest doing that as a followup to get more repos onto Actions first

File: .github/workflows/android.yml (hunk 1 above)
Comment (tyvsmith) [reply .github/workflows/android.yml:31]: This likely gives better output and allows for things like paralllelizing. See https://github.com/uber/AutoDispose/blob/main/.github/workflows/ci.yml

**Comment by jbarr21** [comment]
once the iOS side is completed, you can remove the `.travis.yml` file

**Review by jbarr21 — APPROVED** [review]

**Comment by sanchezz93** [comment]
Added FOSSA issue to follow up

**Review by jbarr21 — APPROVED** [review]

**Pull request merged**